resources/user-agents/browsers/icab/icab-2-x.json

```diff
@@ -29,28 +29,13 @@
         {
           "match": "iCab/#MAJORVER#.#MINORVER#* (#PLATFORM#)",
           "platforms": [
-            "Mac68K", "MacPPC_3", "OSX"
-          ]
-        },
-        {
-          "match": "iCab/#MAJORVER#.#MINORVER#* (#PLATFORM#)",
-          "device": "Windows Desktop",
```

Comment:
iCab doesn't support windows, there's a comment in the `issue-000-invalids.php` fixture that links to the source for this.